### PR TITLE
Paper figs

### PR DESCRIPTION
**Pull request by evevkovacs**
> If you are submitting a PR for a validation test (either adding a new one or updating an existing one), please post a link to the DESCQA web interface for the test you are submitting. 
Many minor updates to configuration files and tests to accommodate changes for the validation paper. Added 2 new data sets for the color-redshift test and the galaxy bias test.

### CODE REVIEW

File: descqa/ColorRedshiftTest.py [hunk 1]
```diff
@@ -108,15 +163,41 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                 fig.colorbar(pc, ax=ax).set_label("Population Density")
             mag1 = re.split('_', mag1_str)[1]  #get filter
             mag2 = re.split('_', mag2_str)[1]  #get filter
+            # plot observations
+            for k,v in self.validation_data.items():
```
Comment (yymao) [inline descqa/ColorRedshiftTest.py:167]: `k` is not used

```suggestion
            for v in self.validation_data.values():
```

File: descqa/ColorRedshiftTest.py [hunk 2]
```diff
@@ -108,15 +163,41 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
...
+                    obs = np.zeros(len(redshift_bins[zmask]))
+                    for n, coeff in enumerate(coeffs):
+                        obs += coeff*redshift_bins[zmask]**(len(coeffs)-1-n)
+
+                    ax.plot(redshift_bins[zmask], obs, color='r', label=v['label'])
+                elif v['format'] == 'data':
+                    if color in v.keys():
+                        zbins = np.asarray(v['bins'])
+                        mean, edge, num = bs(v['z'], v[color], bins=zbins)
+                        std, edge, num = bs(v['z'], v[color], bins=zbins, statistic='std')
```
Comment (yymao) [inline descqa/ColorRedshiftTest.py:181]: `edge` not used
```suggestion
                        mean, _, num = bs(v['z'], v[color], bins=zbins)
                        std, _, num = bs(v['z'], v[color], bins=zbins, statistic='std')
```

File: descqa/BiasVersusRedshift.py [hunk 3]
```diff
@@ -39,19 +65,45 @@ def __init__(self, **kwargs): #pylint: disable=W0231
...
 
         for sample_name, color in zip(self.test_samples, colors):
             sample_corr = corr_data[sample_name]
             sample_label = self.test_sample_labels.get(sample_name)
             sample_th = corr_theory[sample_name]
             ax[0].loglog(sample_corr[0], sample_th, c=color)
-            ax[0].errorbar(sample_corr[0], sample_corr[1], sample_corr[2], marker='o', ls='', c=color,
-                           label=sample_label)
-
+            markers, caps, bars = ax[0].errorbar(sample_corr[0], sample_corr[1], sample_corr[2], marker='o', ls='', c=color,
```
Comment (yymao) [inline descqa/BiasVersusRedshift.py:97]:
```suggestion
            _, caps, bars = ax[0].errorbar(sample_corr[0], sample_corr[1], sample_corr[2], marker='o', ls='', c=color,
```

File: descqa/BiasVersusRedshift.py [hunk 4]
```diff
@@ -63,10 +115,21 @@ def plot_bias_results(self, corr_data, corr_theory, bias, z, catalog_name, outpu
         ax[0].set_ylim(*self.fig_ylim)
         ax[0].set_ylabel(self.fig_ylabel, size=self.font_size)
         ax[0].set_title(title, fontsize='medium')
-        ax[1].plot(z,bias)
+
+        ax[1].errorbar(z, bias, err, marker='o', ls='', label=catalog_name)
+        if not self.observations:
+            ax[1].plot(z, bias)
+        #add validation data
+        for k, v in self.validation_data.items():
```
Comment (yymao) [inline descqa/BiasVersusRedshift.py:123]:
```suggestion
        for v in self.validation_data.values():
```

**Review by yymao — COMMENTED** [review]
removing unused variables

**Comment by evevkovacs** [comment]
@yymao The remaining changes are all minor changes to config files. I also saved all the config files that I used to run the figures for the paper, which I think is worthwhile doing. I asked you to review. Thanks

**Comment by evevkovacs** [comment]
@yymao  I asked for your review. The remaining changes are all minor. They facilitated the production of the figures. Config files for all the figures in the paper are also saved. Thanks

**Review by yymao — APPROVED** [review]
LGTM

**Pull request merged**